functions/imagemagick/main_test.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+
+from mock import MagicMock, patch
+
+import main
+
+
+class DictObject(dict):
```

Comment:
Depending on what you need this for, you might be able to use Python 3's "UserDict".

Reply:
UserDict only works in Python 3 - I'm assuming we don't care about supporting 2.7?

Reply:
Done.